array-destructuring/exercise-1/exercise.js

```diff
@@ -4,7 +4,8 @@ const personOne = {
   favouriteFood: "Spinach",
 };
 
-function introduceYourself(___________________________) {
+
+function introduceYourself({ name, age, favouriteFood } = personOne) {
```

Comment:
Do you need to set a default for the object that is passed in?

Reply:
Thank you for pointing it out, Jack! Just refactored it.